### PR TITLE
Changes to only support Ubuntu 16.04 LTS

### PR DESCRIPTION
**Pull request by pumazi**
There are some lingering Ubuntu 14.10 statements in here. I feel we need to leave them behind to keep cte-dev functional in the interim.

---

:skull: **DO NOT MERGE this pull request** ￼:skull:

This project is manually merged. This pull request is only for review and comment.

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -4,6 +4,23 @@
 # Note, 'local' (the host name in the inventory)
 #   should be assigned ``ansible_connection=local``.
 
+# Ansible requires python2, while Ubuntu >=15 strictly uses Python3.
```
Comment (reedstrm) [inline main.yml]: I installed against a fresh install of 16.04, and did no need this. python is still 2.7 on 16.04.

Comment (pumazi) [inline main.yml]: My install shows the following:

```sh
ubuntu@rubble:~$ python --version
The program 'python' can be found in the following packages:
 * python-minimal
 * python3
Try: sudo apt install <selected package>
ubuntu@rubble:~$ python2 --version
The program 'python2' is currently not installed. You can install it by typing:
sudo apt install python-minimal
ubuntu@rubble:~$ python3 --version
Python 3.5.2
```

Comment (karenc) [inline main.yml]: I also did a fresh install of ubuntu server 16.04.1 (http://releases.ubuntu.com/16.04.1/ubuntu-16.04.1-server-amd64.iso) and it did not have `python` installed.

Comment (reedstrm) [inline main.yml]: I with draw my comment. ISTR I did in fact select something in the 'tasksel' setp of the install, I bet that dragged it in.

**Pull request merged**